### PR TITLE
Paul/paybis sell

### PR DESCRIPTION
**Pull request by paullinator**
### CHANGELOG

Does this branch warrant an entry to the CHANGELOG?

- [x] Yes
- [ ] No

### Dependencies

 none

### Requirements

If you have made **any** visual changes to the GUI. Make sure you have:

- [x] Tested on iOS device
- [ ] Tested on Android device
- [ ] Tested on small-screen device (iPod Touch)
- [ ] Tested on large-screen device (tablet)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205654034220688

### CODE REVIEW

File: src/plugins/gui/providers/paybisProvider.ts [hunk 1]
```diff
@@ -93,48 +155,44 @@ const asQuote = asObject({
...
+  amount: asString
 })
 
 type PaymentMethodId = ReturnType<typeof asPaymentMethodId>
-type PaybisPairs = ReturnType<typeof asPaybisPairs>
+type PaybisBuyPairs = ReturnType<typeof asPaybisBuyPairs>
+type PaybisSellPairs = ReturnType<typeof asPaybisSellPairs>
 
-let paybisPairs: PaybisPairs | undefined
+interface InittializePairs {
```
Comment (Jon-edge) [inline src/plugins/gui/providers/paybisProvider.ts:178]: Spelling

```suggestion
interface InitializePairs {
```

File: src/plugins/gui/providers/paybisProvider.ts [hunk 2]
```diff
@@ -75,15 +97,55 @@ const asPaymentMethodPairs = asObject({
...
 })
 
+const asSellPair = asObject({
+  fromAssetId: asString,
+  to: asArray(asString)
+})
+
+const asSellPaymentMethodPairs = asObject({
+  name: asMaybe(asPaymentMethodId),
+  // displayName: asString,
```
Comment (Jon-edge) [inline src/plugins/gui/providers/paybisProvider.ts:111]: Doesn't look like we're using this, is there a plan to incorporate this later?

**Review by Jon-edge — APPROVED** [review]

File: src/plugins/gui/providers/paybisProvider.ts (hunk 2 above)
Comment (paullinator) [reply src/plugins/gui/providers/paybisProvider.ts:111]: It is being used. Otherwise it would be a lint error

Comment (paullinator) [reply src/plugins/gui/providers/paybisProvider.ts:111]: It is being used. Otherwise it would be a lint error

Comment (paullinator) [reply src/plugins/gui/providers/paybisProvider.ts:111]: It is being used. Otherwise it would be a lint error

**Pull request merged**